### ISSUE TITLE
[Feedback] Inconsistencies in components use to render HTML tags

### ISSUE THREAD

**Issue by rcp522** [aws-amplify/docs#4756]
**Page**: [`/start/getting-started/auth/q/integration/[integration]`](https://docs.amplify.aws/start/getting-started/auth/q/integration/react)

**Feedback**:

With React you want to componentize as much as possible to increase applications re-render performance. I can see that the Amplify UI does have Components such as Button, View, Text which can be used instead of button, div and p tags in html.

In this linked docs we can see the example sometimes uses the Amplify UI component and other times use the native html component.  As an example, line 49 has the following ` <Button onClick={signOut} style={styles.button}>Sign out</Button>` whereas line 63 has `<button style={styles.button} onClick={addTodo}>Create Todo</button>` .

Below is my suggestion
```Javascript
/* src/App.js */
import React, { useEffect, useState } from 'react'
import { Amplify, API, graphqlOperation } from 'aws-amplify'
import { createTodo } from './graphql/mutations'
import { listTodos } from './graphql/queries'
import {
  withAuthenticator,
  Button,
  Heading,
  View,
  Text,
  TextField,
} from "@aws-amplify/ui-react";
import '@aws-amplify/ui-react/styles.css';

import awsExports from "./aws-exports";
Amplify.configure(awsExports);

const initialState = { name: '', description: '' }

const App = ({ signOut, user }) => {
  const [formState, setFormState] = useState(initialState)
  const [todos, setTodos] = useState([])

  useEffect(() => {
    fetchTodos()
  }, [])

  function setInput(key, value) {
    setFormState({ ...formState, [key]: value })
  }

  async function fetchTodos() {
    try {
      const todoData = await API.graphql(graphqlOperation(listTodos))
      const todos = todoData.data.listTodos.items
      setTodos(todos)
    } catch (err) { console.log('error fetching todos') }
  }

  async function addTodo() {
    try {
      if (!formState.name || !formState.description) return
      const todo = { ...formState }
      setTodos([...todos, todo])
      setFormState(initialState)
      await API.graphql(graphqlOperation(createTodo, {input: todo}))
    } catch (err) {
      console.log('error creating todo:', err)
    }
  }

  return (
    <View style={styles.container}>
      <Heading level={1}>Hello {user.username}</Heading>
      <Button onClick={signOut} style={styles.button}>Sign out</Button>
      <Heading level={2}>Amplify Todos</Heading>
      <TextField
        onChange={event => setInput('name', event.target.value)}
        style={styles.input}
        value={formState.name}
        placeholder="Name"
      />
      <TextField
        onChange={event => setInput('description', event.target.value)}
        style={styles.input}
        value={formState.description}
        placeholder="Description"
      />
      <Button style={styles.button} onClick={addTodo}>Create Todo</Button>
      {
        todos.map((todo, index) => (
          <View key={todo.id ? todo.id : index} style={styles.todo}>
            <Text style={styles.todoName}>{todo.name}</p>
            <Text style={styles.todoDescription}>{todo.description}</p>
          </View>
        ))
      }
    </View>
  )
}

const styles = {
  container: { width: 400, margin: '0 auto', display: 'flex', flexDirection: 'column', justifyContent: 'center', padding: 20 },
  todo: {  marginBottom: 15 },
  input: { border: 'none', backgroundColor: '#ddd', marginBottom: 10, padding: 8, fontSize: 18 },
  todoName: { fontSize: 20, fontWeight: 'bold' },
  todoDescription: { marginBottom: 0 },
  button: { backgroundColor: 'black', color: 'white', outline: 'none', fontSize: 18, padding: '12px 0px' }
}

export default withAuthenticator(App);
```

**Comment by siegerts** [comment]
Hi @rcp522, thanks for opening. There is an extra section, https://docs.amplify.aws/start/getting-started/auth/q/integration/react/#bonus-use-amplify-ui-primitives, that swaps out rest of the native tags for the Amplify primitives in that example. Does that help?

cc @rachelnabors

**Comment by rcp522** [comment]
Hi @siegerts Yes it does. I would also suggest replacing the `div` tags with `View` tags

**Comment by rachelnabors** [comment]
What a great idea! If you create a PR to do that, we will merge it in!

**Comment by cwomack** [comment]
Merged #4800 for update to docs to resolve this issue with community submission from @rcp522.  We appreciate your submission and I will will close this issue out now!
